database/sql/V048__create_settlement_rewards.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE settlements
+  ADD COLUMN id integer default NULL,
+  ADD COLUMN execution_cost integer default NULL;
+
+CREATE TABLE settlement_rewards (
+  settlement_id integer,
+  simulated_surplus integer NOT NULL,
+  simulated_fee integer NOT NULL,
```

Comment:
Ok - so it seems the problem everyone has with this is that we should include "observed" values and that they belong in a different table. I will make the suggested changes, but then the name of this table will change.